docs/public/sandbox.js

```diff
@@ -0,0 +1,19 @@
+import * as R from "https://esm.sh/remeda";
+
+window.R = R;
+
+const sansSerif = "font-family: sans-serif; font-size: 16px";
+const monospace = "font-family: monospace; font-size: 16px";
```

Comment:
global consts should be UPPER_CASE
```suggestion
const SANS_SERIF = "font-family: sans-serif; font-size: 16px";
const MONOSPACE = "font-family: monospace; font-size: 16px";
```